src/kbc_modules/cc_kbc/mod.rs

```diff
@@ -146,7 +151,7 @@ impl Kbc {
 
         let challenge = self
             .http_client()
-            .post(format!("{kbs_uri}/{KBS_URL_PREFIX}/auth"))
+            .post(format!("{kbs_uri}{KBS_URL_PREFIX}/auth"))
```

Comment:
Why we remove `/` here?

Reply:
Does this mean the `kbs_uri` in `fn new(kbs_uri: String) -> Result<Kbc>` must be `http://.../` but can not be `http://...` ?

Reply:
Oh, I saw the test cases in the unit tests, and they worked properly. There's no problem doing this. Sorry!